src/components/SingleFeature/index.css

```diff
@@ -1,11 +1,25 @@
-.imgclass {
-  display: flex;
-  flex-flow: row wrap;
-  align-items: center;
-  justify-content: space-around;
+.blogContainer {
+  text-align: justify;
+  margin: 100px auto;
+  border: #6c567b;
 }
 
-.featuresImg {
-  max-width: 100%;
-  height: 100%;
+.roundImage {
+  width: 100%;
+  height: auto;
+  align-self: center;
+  object-fit: contain;
+  margin-top: 1rem;
+}
+
+.blogTitle {
+  margin-bottom: 1em;
+  color: #6c567b;
+  font-size: 30px;
+}
+
+.blogBody {
+  color: #4a474d;
+  font-size: 20px;
+  line-height: 150%;
```

Comment:
Please, use variables for colors and rem instead of px

Reply:
revised